ddtrace/contrib/internal/avro/patch.py

```diff
@@ -0,0 +1,81 @@
+import avro
+import wrapt
+
+from ddtrace import config
+from ddtrace.internal.utils.wrappers import unwrap
+from ddtrace.pin import Pin
+
+from .schema_iterator import SchemaExtractor
+
+
+config._add(
+    "avro",
+    dict(),
+)
+
+
+def get_version():
+    # type: () -> str
+    return getattr(avro, "__version__", "")
+
+
+def patch():
+    """Patch the instrumented methods"""
+    if getattr(avro, "_datadog_patch", False):
+        return
+    avro._datadog_patch = True
+
+    _w = wrapt.wrap_function_wrapper
+
+    _w("avro.io", "DatumReader.read", _traced_deserialize)
+    _w("avro.io", "DatumWriter.write", _traced_serialize)
+    Pin(service=None).onto(avro.io.DatumReader)
+    Pin(service=None).onto(avro.io.DatumWriter)
```

Comment:
Do we need to use `Pin` here? If we do, should we specify the service name here instead of leaving it as `None`?